AxonDeepSeg/segment.py

```diff
@@ -298,8 +298,8 @@ def main(argv=None):
         psm = float(args["sizepixel"])
     else:
         psm = None
-    path_target_list = [Path(path) for path in args["imgpath"]]
-    new_path = args["model"]
+    path_target_list = [Path(p) for p in args["imgpath"]]
+    new_path = Path(args["model"])
```

Comment:
Actually, looks like this line change caused failing tests. Feel free to revert.

Reply:
Either revert, or maybe it needs the same format as above (e.g. `new_path = [Path(p) for p in args["model"]]`)?